go/vt/tabletserver/table_info.go

```diff
@@ -32,6 +41,13 @@ func NewTableInfo(conn *DBConn, tableName string, tableType string, comment stri
 	if err != nil {
 		return nil, err
 	}
+	if strings.Contains(comment, "vitess_sequence") {
+		err = ti.initSequence(conn)
```

Comment:
this doesn't feel right, if vttablet is restarted after having handed out a couple values from the current batch, it will hand out the same values again, won't it?